src/bindable-property.js

```diff
@@ -248,3 +292,18 @@ export class BindableProperty {
     observer.selfSubscriber = selfSubscriber;
   }
 }
+
+/**
+ * @private
+ * Used for avoid creating mapping function multiple times
+ * @param {Element} element 
+ * @param {string} propertyName 
+ * @param {any} newValue 
+ */
+function propToAttr(element, propertyName, newValue) {
```

Comment:
It seems to me this doesn't take into account the camelCase to kebab-case between properties and html attributes? 

Also in HTML a few weird properties do not map directly to same-name attribute. I think we have a kind of map somewhere in Aurelia that converts some of those so that it "just work" when binding HTML attributes, shouldn't we re-use it here so that it just works for those?

Should this API interact with the `attribute` binding option? It allows users to map a property to a different name attribute, wouldn't they expect `reflectToAttribute` to take that into account? Seems more intuitive.

Reply:
No camelCase to kebab-case is a bad mistake. It should be fixed.

Code for the special pairs belong to `templating-binding`, should I copy it over ? I'm not sure how to handle this, because props of custom element in Aurelia, donot necessarily map to attribute like built-in props.

Reply:
You are right. This is for custom attributes, it has nothing to do with built-in HTML attributes.
Forget my remark about the map ☺️

Reply:
For the `attribute`, it sounds really good. Maybe i'll go for it and update the PR doc.